src/actions/index.js

```diff
@@ -0,0 +1,29 @@
+let taskId = 0;
+
+export const addTask = (title, description) => ({
+  type: "ADD_TASK",
+  id: taskId++,
```

Comment:
These actions are good, but one improvement that i forget to mention you its that the actions always are objects with two properties, type and payload, you already have the type.
In the payload, you can put everything that you can pass to the reducer..
In this example would be like:
`type: "ADD_TASK",`
`payload:{ `
    `id: taskId++`
`}`

the same with the other actions..